### ISSUE TITLE
import a journey from blablacar.fr

### ISSUE THREAD

**Issue by maxbellec** [covoiturage-libre/covoiturage-libre#328]
### Feature request
Import a journey from blablacar.fr

### Ideas for implementing the new functionality  
I'm not familiar with Ruby but I'd be happy to implement a Python scripts that outputs for example in JSON format with the field names corresponding to those used in CVL.

### Pros
If we can make it work so that one can import a journey from BBC to CVL in 30 seconds, I believe it could attract many more journey.

### Cons
The script would have to be updated each time one of the website (BBC or CVL) gets updated.

I believe we can implement the feature with the current layout for both websites. If many people use it, it will be worth it to maintain.

You probably already had a debate about it, but I didn't find anything on Github.

**Comment by nicolasleger** [comment]
I thought about that just from the trip link from this platform (or could implement another).

**Comment by nicolasleger** [comment]
By adding a trip link input in creation form.
Then if filled fetch the link to retrieve the metadata and transform it in CVL format.
Redisplay the creation form for the person to valid data.

**Comment by maxbellec** [comment]
Yes exactly what I'm thinking. And probably add the few missing fields that we cannot get from BBC.

**Comment by nicolasleger** [comment]
They also have an API https://dev.blablacar.com/api-explorer/search-api/versions/1.0//
